packages/ocsipersist/ocsipersist.1.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors:      "The Ocsigen team <dev@ocsigen.org>"
+maintainer:   "Jan Rochel <jan@besport.com>"
+license:      "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis:     "Persistent key/value storage (for Ocsigen) using multiple backends"
+description:  "This is an virtual library defining a unified frontend for a number of key/value storage implementations. Ocsipersist is used pervasively in Eliom/Ocsigen to handle sessions and references. It can be used as an extension for ocsigenserver or as a library. Implementations of the following backends currently exist: DBM, PostgreSQL, SQLite."
+
+homepage: "https://github.com/ocsigen/ocsipersist"
+bug-reports: "https://github.com/ocsigen/ocsipersist/issues"
+dev-repo: "git+https://github.com/ocsigen/ocsipersist.git"
+build:   [ "dune" "build" "-p" name "-j" jobs ]
+
+depends: [
+  "dune" {>= "2.9"}
+  "lwt" {>= "4.2.0"}
+  "ocsigenserver" {>= "3.0.0"}
+  "ocsipersist-lib"
```

Comment:
```suggestion
  "ocsipersist-lib" {>= "1.1.0" & < "1.2.0"}
```
To match the other packages.

Also previous versions of ocsipersist also require the same kind of constraints (see CI). Could you add them?

Reply:
Thanks, I will.